File: codeExamples/week11/spring24_help_reviews-main/client/src/models/Report.js
export class Report {
  constructor(data) {
    this.id = data.id
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)
    this.title = data.title
    this.body = data.body
    this.imgUrl = data.imgUrl || ''
    this.restaurantId = data.restaurantId
    this.creatorId = data.creatorId
    this.creator = data.creator
  }
}

File: codeExamples/week11/spring24_help_reviews-main/server/Models/Report.cs
namespace help_reviews.Models;

public class Report : RepoItem<int>
{
  public string Title { get; set; }
  public string Body { get; set; }
  public string ImgUrl { get; set; }
  public int RestaurantId { get; set; }
  public string CreatorId { get; set; }
  public Profile Creator { get; set; }
}